kolibri/core/content/utils/channels.py

```diff
@@ -121,17 +121,17 @@ def get_channels_for_data_folder(datafolder):
             continue
         channel_data = {
             "path": path,
-            "id": channel.id,
-            "name": channel.name,
-            "description": channel.description,
-            "tagline": getattr(channel, "tagline", ""),
-            "thumbnail": channel.thumbnail,
-            "version": channel.version,
-            "root": channel.root_id,
-            "author": channel.author,
-            "last_updated": getattr(channel, "last_updated", None),
-            "lang_code": getattr(channel, "lang_code", None),
-            "lang_name": getattr(channel, "lang_name", None),
+            "id": channel["id"],
+            "name": channel["name"],
+            "description": channel["description"],
+            "tagline": channel.get("tagline", ""),
+            "thumbnail": channel["thumbnail"],
+            "version": channel["version"],
+            "root": channel["root_id"],
+            "author": channel["author"],
+            "last_updated": channel.get("last_updated"),
+            "lang_code": channel.get("lang_code"),
+            "lang_name": channel.get("lang_name"),
```

Comment:
Is the inconsistency in getter style because some of these are supposed to throw value errors and others are supposed to return `None` if the items doesn't exist?

(Mainly just curious if this is part of an implicit required/optional API of some kind)

Reply:
As it was 'like that when I got here' at this point, I am inferring - but I think these use of `get` versus direct key access is a result of newer fields being added that might be undefined in historic channel DB schema.